### PR TITLE
5 packages from ocsigen/ocsipersist

### PR DESCRIPTION
**Pull request by jrochel**
This pull-request concerns:
-`ocsipersist.1.0.1`: Persistent key/value storage (for Ocsigen) using multiple backends
-`ocsipersist-dbm.1.0.1`: Persistent key/value storage (for Ocsigen) using DBM
-`ocsipersist-lib.1.0.1`: Persistent key/value storage (for Ocsigen) - support library
-`ocsipersist-pgsql.1.0.1`: Persistent key/value storage (for Ocsigen) using PostgreSQL
-`ocsipersist-sqlite.1.0`: Persistent key/value storage (for Ocsigen) using SQLite

---
* Homepage: https://github.com/ocsigen/ocsipersist
* Source repo: git+https://github.com/ocsigen/ocsipersist.git
* Bug tracker: https://github.com/ocsigen/ocsipersist/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/ocsipersist-sqlite/ocsipersist-sqlite.1.0/opam [hunk 1]
```diff
@@ -20,9 +20,9 @@ depends: [
   "sqlite3"
 ]
 url {
-  src: "https://github.com/jrochel/ocsipersist/archive/1.0.tar.gz"
+  src: "https://github.com/ocsigen/ocsipersist/archive/1.0.1.tar.gz"
   checksum: [
-    "md5=d07413b0b7802b88e7bf7d52d141a48d"
-    "sha512=4205748b93a9f74e8e2c1f7ddfc4a2a90f054abadeda5a0cf020eeabbf4d429d57b9b8a9d1a5b6fed733f48e414bbcdef88893e829c3925c755abea7fe5b67a0"
+    "md5=ffbd09c9fe79c0fd9c5013dce0975a7c"
+    "sha512=7b038f35fb93b9983642cbff2f4069cacd5fe9f20118e3cb6aca685a7e2f16f320b73daa13b39dd45429fb6e884699dc7cfca9cfc459c8db86baf4846c9fb7e9"
```
Comment (kit-ty-kate) [inline packages/ocsipersist-sqlite/ocsipersist-sqlite.1.0/opam:26]: This changes the old version to 1.0.1. Could you revert this change?

Comment (jrochel) [reply packages/ocsipersist-sqlite/ocsipersist-sqlite.1.0/opam]: Thanks. Done!

**Comment by kit-ty-kate** [comment]
Thanks!
